test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -41,6 +41,19 @@ const filename = path.join(common.tmpDir, 'sync-write-stream.txt');
   assert.strictEqual(fs.readFileSync(filename).equals(chunk), true);
 }
 
+// Throws if data is not of type Buffer.
+{
+  const stream = new SyncWriteStream(1);
+  common.expectsError(() => {
+    stream._write(42, null, function() {});
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE',
+    type: TypeError,
+    message: 'The "data" argument must be of type Buffer without null bytes.' +
```

Comment:
Umm... If I'm not mistaken, the message here will not match the error it actually throws.
I think it should be `'The "data" argument must be of type Buffer. Received type number'`.

Reply:
Yes, that's probably why the test is failing, you just need to change this to match the actual error message.

Reply:
Fixed! 🔧